videostore/utils.go

```diff
@@ -253,31 +254,31 @@ func sortFilesByDate(files []fileWithDate) {
 }
 
 // extractDateTimeFromFilename extracts the date and time from the filename.
+// Returns UTC time regardless of filename format:
+// - Unix timestamp filenames: directly converts to UTC time
+// - Datetime format filenames (YYYY-MM-DD_HH-mm-ss): parses as local time, converts to UTC
 func extractDateTimeFromFilename(filePath string) (time.Time, error) {
```

Comment:
can we change all the function parse strings into time.Time to return time.Time in the timezone the string was in:
If a `"2006-01-02_15-04-05"` format string, it should be in local time, if a UNIX timestamp, should be in UTC time.

The caller can then call a `UTC()` to normalize it.

Reply:
Fixed